content/quickstart-docker.ps1

```diff
@@ -16,7 +16,7 @@
 
 # Default values for SkyWalking versions
 $SW_VERSION = "9.7.0"
-$SW_BANYANDB_VERSION = "0.5.0"
+$SW_BANYANDB_VERSION = "0.6.0"
```

Comment:
Can I update this line @wu-sheng

Reply:
You should update both. 10.0.0 + 0.6. But you should verify logically first through netlify hosted website.

Reply:
@kezhenxu94 Is this the only place to update?

Reply:
I would like leaving it in following update commits.